### PR TITLE
Update load tests

### PR DESCRIPTION
**Pull request by Katka92**
### What does this PR do?
Stabilize workspace creation, change approach to work with https and self-signed certificate.
Add a script that can sum up a few known failures and ease the investigation of results.

### CODE REVIEW

File: tests/e2e/tests/load_test/LoadTest.spec.ts [hunk 1]
```diff
@@ -15,27 +15,34 @@ import { ProjectTree } from '../../pageobjects/ide/ProjectTree';
...
-import { TestConstants } from '../..';
+import { TestConstants, NameGenerator } from '../..';
 const driverHelper: DriverHelper = e2eContainer.get(CLASSES.DriverHelper);
 const ide: Ide = e2eContainer.get(CLASSES.Ide);
 const projectTree: ProjectTree = e2eContainer.get(CLASSES.ProjectTree);
-const factoryUrl: string = `${TestConstants.TS_SELENIUM_BASE_URL}/f?url=https://gist.githubusercontent.com/Katka92/fe747fa93b8275abb2fb9f3803de1f0f/raw`;
 const cheLoginPage: ICheLoginPage = e2eContainer.get<ICheLoginPage>(TYPES.CheLogin);
 const testWorkspaceUtils: TestWorkspaceUtil = e2eContainer.get<TestWorkspaceUtil>(TYPES.WorkspaceUtil);
+const workspaceName: string = NameGenerator.generate('wksp-test-', 5);
+const workspacePrefixUrl: string = `${TestConstants.TS_SELENIUM_BASE_URL}/dashboard/#/ide/TestConstants.TS_SELENIUM_USERNAME/`;
```
Comment (rhopp) [inline tests/e2e/tests/load_test/LoadTest.spec.ts]: Aren't you missing ${} in here?

File: tests/performance/sum_up_failures.sh [hunk 2]
```diff
@@ -0,0 +1,155 @@
+#!/bin/bash
```
Comment (rhopp) [inline tests/performance/sum_up_failures.sh:1]: Missing license 

Also please add some doc to the header about what this script does nad how can one use it.

**Review by rhopp — CHANGES_REQUESTED** [review]

File: tests/e2e/tests/load_test/LoadTest.spec.ts (hunk 1 above)
Comment (ScrewTSW) [reply tests/e2e/tests/load_test/LoadTest.spec.ts]: yes, I think the ${} is missing as well

**Review by ScrewTSW — CHANGES_REQUESTED** [review]

File: tests/e2e/tests/load_test/LoadTest.spec.ts (hunk 1 above)
Comment (Katka92) [reply tests/e2e/tests/load_test/LoadTest.spec.ts]: I've fixed that also in GitSsh test flow.

**Review by rhopp — APPROVED** [review]

**Pull request merged**